.buildkite/scripts/lifecycle/post_build.sh

```diff
@@ -9,7 +9,7 @@ if [[ "${GITHUB_BUILD_COMMIT_STATUS_ENABLED:-}" != "true" ]]; then
   "$(dirname "${0}")/commit_status_complete.sh"
 fi
 
-ts-node "$(dirname "${0}")/ci_stats_complete.ts"
```

Comment:
Sorry, I saw the earlier discussion but haven't had a chance to chime in.  

Mind if we revert this so we can move discussion to another issue?  Merging in latest should resolve the issue.  This can be prevented by not cancelling builds in kibana-on-merge.

CI stats here is used as a smoke check to compare metrics and verify there aren't any unexpected bundle changes.  In this case it's a non issue, but I think we need explicit agreement that we're okay with that happening before proceeding.

Reply:
I see, I will revert this from the PR. Technically, I can also push changes to the branch directly once PR is reviewed by respective teams. So for now, it's ok if the build is failed. When this is finally merged in `main` through another PR, I believe the CI will pass.

Reply:
Thank you!